mat/eigen_test.go

```diff
@@ -28,17 +28,39 @@ func TestEigen(t *testing.T) {
 				0, 0, 1,
 			}),
 			values: []complex128{1, 1, 1},
-			left: NewDense(3, 3, []float64{
+			left: NewCDense(3, 3, []complex128{
 				1, 0, 0,
 				0, 1, 0,
 				0, 0, 1,
 			}),
-			right: NewDense(3, 3, []float64{
+			right: NewCDense(3, 3, []complex128{
 				1, 0, 0,
 				0, 1, 0,
 				0, 0, 1,
 			}),
 		},
+		{
+			// Values compared with numpy.
+			a: NewDense(4, 4, []float64{
+				0.9025, 0.025, 0.475, 0.0475,
+				0.0475, 0.475, 0.475, 0.0025,
+				0.0475, 0.025, 0.025, 0.9025,
+				0.0025, 0.475, 0.025, 0.0475,
+			}),
+			values: []complex128{1, 0.7300317046114154, complex(-0.1400158523057075, 0.452854925738716), complex(-0.1400158523057075, -0.452854925738716)},
```

Comment:
`complex(-0.1400158523057075, 0.452854925738716)` can be written as `-0.1400158523057075+0.452854925738716i`, and the others.

Reply:
Done